content/onionservices/onionservices-5/contents.lr

```diff
@@ -0,0 +1,26 @@
+model: _question
+---
+title: What do the different onion icons in the address bar mean?
+---
+seo_slug: address-bar-onion-icons
+---
+description:
+When browsing an Onion Service, Tor Browser displays different onion icons in the address bar indicating the security of the current webpage.
+
+![Image of a green onion](/static/images/green-onion.png)
+A green onion means:
+- The Onion Service is served over HTTP, or HTTPS with a self-signed certificate.
+
+![Image of a green onion with a lock](/static/images/green-onion-with-lock.png)
+A green onion with a lock means:
+- The Onion Service is served over HTTPS with a CA-Issued certificate.
+
+![Image of a green onion with a warning triangle](/static/images/green-onion-with-warning.png)
+A green onion with a warning triangle means:
+- The Onion Service is served over HTTPS with a CA-Issued certificate.
+- The webpage contains subresources served over HTTPS.
```

Comment:
Does this mean that the subresources are served over HTTPS without a CA-Issued certificate but a self-signed one instead?

Reply:
I was wondering that myself. I think it's like a [mixed content](https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content) warning only here third-party HTTPS content is being mixed with Onion Service content. And I believe that only a CA-Issued cert would be trusted in that scenario (chain of trust required since it's outside the Onion Service?).

I could be wrong though! Anyone have other ideas about how this works?

Reply:
[More info](https://docs.google.com/document/d/1bPrNLIl7Qy-sA7aTfElu80Xk2eXzTfH_5BGTOUDK8XU/edit) on these states (Thanks Geko!). Looks like this one is at the bottom of the document.

Reply:
I don't see any green onion with a warning triangle on there 🤔

Reply:
Yeah, looks like the icon changed a bit since that document. I think the case is "An onion embeds HTTPS". That document doesn't say what type of certificate either unfortunately.

Reply:
Might be worth revising the doc and making it clearer I think

Reply:
Looks like these are the final icons included in the source https://docs.google.com/document/d/1bPrNLIl7Qy-sA7aTfElu80Xk2eXzTfH_5BGTOUDK8XU/edit